php_fpm/test_php_fpm.py

```diff
@@ -96,9 +96,6 @@ def test_status(self):
         for mname in metrics:
             self.assertMetric(mname, count=1, tags=expected_tags)
 
-        self.assertMetric('php_fpm.processes.idle', count=1, value=0)
```

Comment:
The `verify` option already defaults to `True` and we don't make any changes to the test configuration, so I didn't expect changes to the tests. Why aren't these metrics collected anymore?

Reply:
The metrics are collected and tested for presence in the loop above.
Testing for the exact value of `idle` was flaky, not sure why but IMO the value it's not important.

Reply:
I think you're right, the value is probably not relevant.